Robert Stoelhorst/week01/day_04/functions/homework.js

```diff
@@ -0,0 +1,69 @@
+// Geometry Function Lab
+// Part 1, Rectangle
+// Given the following a rectangle object like the one below, write the following functions which accept a rectangle object as an argument:
+//
+// isSquare - Returns whether the rectangle is a square or not
+// area - Returns the area of the rectangle
+// perimeter - Returns the perimeter of the rectangle
+
+const areaOfRectangle = function (x, y) {
+  const area = x * y;
+  const perimeter = x * 2 + y * 2;
+  if (x === y)
+return (` This is a square and the area of this square is ${ area }cm2 and it's perimeter is ${ perimeter }cm`);
+  if (x !== y)
+return (` The area of this rectangle is ${ area }cm2 and it's perimeter is ${ perimeter }cm`)
+}
+  areaOfRectangle(10, 10)
+
+  //
+  // Part 2, Triangle
+  // Given the following a triangle object like the one below, write the following functions which accept a triangle object as an argument:
+  //
+  // isEquilateral - Returns whether the triangle is equilateral or not
+  // isIsosceles - Returns whether the triangle is isosceles or not
+  // area - Returns the area of the Triangle
+  // isObtuse - Returns whether the triangle is obtuse or not
+
+  const triangleTypesArea = function (x, y, z) {
+    // const areaEquilateral =
+    // const areaIsoscelese =
+    // // const areaObtuse = ((y * y))
+    if (x === y && y === z)
+    return (`This is an Equilateral triangle and its area is `);
+    else if ((x * x) > ((y * y) + (z * z)))
+    return (`This is an Obtuse triangle and its area is `);
+    else if (x < y && y === z)
+    return (`This is an Isoscelese triangle and its area is `);
+    else
+    return (`This has gone beyond 180 degrees its definatly not a triangle`)
+  }
```

Comment:
same as comment above about indentation. Also please make sure you include the { } in your if/elseif/else statements. You will run into issues down the track if you don't get into the habit of including these.